STEERING-COMMITTEE.md

```diff
@@ -162,13 +117,157 @@ There are two ways to raise issues to the steering committee for decision:
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
 
+## Composition
+
+The steering committee will have 7 seats.  Five of the seats will be
+Contributing seats, open to any project contributor.  Two of the seats will be
+End-User seats, open only to active end-user community members.  See Candidate
+Eligibility for definitions of these categories.
+
+Steering Committee members will serve for 2-year terms, staggered in order to
+preserve continuity.  Every year either two or three contributor seats will be
+elected, and one end user seat will be elected.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections shall be held in October or November of each year.
+Six weeks or more before the election, the Steering Committee shall appoint
```

Comment:
Continued from my note above: I'd recommend against "shall appoint", which is a future tense. I'd suggest "appoints".

If you keep future tenses, I'd use "will" over "shall". The latter is common in legalese, which can make things less approachable.

Reply:
Ack, incorporating now